src/realm/util/sha_crypto.cpp

```diff
@@ -126,6 +137,35 @@ void message_digest(const EVP_MD* digest_type, const char* in_buffer, size_t in_
         throw realm::util::runtime_error("EVP_DigestFinal_ex() failed");
 }
 #endif
+
+#if REALM_USE_BUNDLED_SHA2
```

Comment:
This appears to never be defined to 0 so it'll produce -Wundef warnings.

Reply:
Ah, good point! I'll switch to `#if defined()` instead.